src/main/java/lrgs/db/LrgsDatabaseThread.java

```diff
@@ -156,27 +157,22 @@ private synchronized void attemptConnect()
 		lastConnectAttempt = System.currentTimeMillis();
 
 		// Get username & password
-		String username = "lrgs_adm";
-		String password = "";
-		String authFileName = EnvExpander.expand("$LRGSHOME/.lrgsdb.auth");
-		UserAuthFile authFile = new UserAuthFile(authFileName);
+		String username = "lrgs_adm";		
```

Comment:
Is this default user documented anywhere? I'm not sure it actually survives the following code.

Reply:
hmm, I'm going to have to check that again. I don't think I created it, just shifted it around. But perhaps now is a good time to clean up that initial user and password generation.

Reply:
you're write, it doesn't survive the logic. I think it might have originally but i've gone ahead and just removed it.